part3.asm

```diff
@@ -0,0 +1,36 @@
+# Author: ENYIL PADILLA
+# Assignment: PA1
+# Date: 2/11/15
+
+############################### TODO MEMORY VARIABLES ###############################
+.data
+total: .word 0:7
+msg: .asciiz "Total sum odd numbers between 1 and 15: "
+
+############################### TODO ACTUAL PROGRAM ###############################
+.text
+li $s1, 15	#Starts at 15
+li $s2, 1	#alternates between sums
+li $s3, 0	#keep count of the sum
+la $s4, total 	#loads memory for total numbers
+while:
+     bgtz $s2, sum	#greater than zero then jump to sum
+     mul $s2, $s2, -1	 #sum next time
```

Comment:
Your implementation is good and works well, though if you wanted to be as efficient as possible you could just decrement by 2 to avoid the two multiplications and one of the branches, but this is very good work.